README.md

```diff
@@ -10,7 +10,7 @@ Tools which can find a range of 'unknown' vulnerabilities on any websites.
 | Main Site | Last Commit | Committers | Stars |
 | --- | --- | --- | --- |
 | [ZAP](https://www.zaproxy.org) | [![Last Commit](https://img.shields.io/github/last-commit/zaproxy/zaproxy)](https://github.com/zaproxy/zaproxy/commits) | [![Contributors](https://img.shields.io/github/contributors/zaproxy/zaproxy)](https://github.com/zaproxy/zaproxy/graphs/contributors) | [![Stars](https://img.shields.io/github/stars/zaproxy/zaproxy)](https://github.com/zaproxy/zaproxy/stargazers) |
-|  - ZAP Extensions | [![Last Commit](https://img.shields.io/github/last-commit/zaproxy/zap-extensions)](https://github.com/zaproxy/zap-extensions/commits) | [![Contributors](https://img.shields.io/github/contributors/zaproxy/zap-extensions)](https://github.com/zaproxy/zap-extensions/graphs/contributors)| [![Stars](https://img.shields.io/github/stars/zaproxy/zap-extensions)](https://github.com/zaproxy/zap-extensions/stargazers) |
+| [ZAP Extensions](https://github.com/zaproxy/zap-extensions) | [![Last Commit](https://img.shields.io/github/last-commit/zaproxy/zap-extensions)](https://github.com/zaproxy/zap-extensions/commits) | [![Contributors](https://img.shields.io/github/contributors/zaproxy/zap-extensions)](https://github.com/zaproxy/zap-extensions/graphs/contributors)| [![Stars](https://img.shields.io/github/stars/zaproxy/zap-extensions)](https://github.com/zaproxy/zap-extensions/stargazers) |
```

Comment:
The "-" is there to indent the repo and make it a bit clearer that it is part of zaproxy. That was the reason I didnt link to it as well, but I could be persuaded otherwise ;)
Note that 'Nuclei Templates' further down the page is similar.

Reply:
That makes sense